src/app/counter/Button/Button.tsx

```diff
@@ -0,0 +1,12 @@
+import "./button.css";
+
+const CounterButton = (props: { onClick: () => void; color: string }) => {
+  return (
```

Comment:
doesn't really matter here, but for bigger files, might be good to destructure props at the beginning of the component